### ISSUE TITLE
How to uninstall Bazel from Windows?

### ISSUE THREAD

**Issue by sahiljuneja** [bazelbuild/bazel#12860]
### Description of the problem / feature request:

I used Bazel to install Tensorflow (from source) a few months ago. But I haven't really used it in quite a while and wanted to uninstall it. 

Except, I don't know how to. It's taking up about 20GB of space on my system and I want to free it up. 

I have a bazel.exe file in my system (I had a folder where I kept track of what I had to install to get a project running which required Tensorflow) so I think I probably used the Bazel binary from GitHub to install it.

Since it's on Windows, I don't want to mess things up by trying something out on my own so looking for help.

### What operating system are you running Bazel on?

Windows 10

### What's the output of `bazel info release`?

`ERROR: The 'info' command is only supported from within a workspace (below a directory having a WORKSPACE file).`

I ran `bazel version` and the build label is `0.26.1`.

###  Have you found anything relevant by searching the web?

I have found threads/posts related to removing it on linux, but nothing on Windows as of now.

**Comment by sahiljuneja** [comment]
Ok, so the ~20GB of space was inside a folder `bazel-out` and almost all of it was being taken by Tensorflow files. 

`bazel-out` contains the build outputs as per https://docs.bazel.build/versions/master/user-manual.html#clean which can be cleaned using `build clean`. 

As per https://www.tensorflow.org/install/source_windows#build_the_package

> Although it is possible to build both CUDA and non-CUDA configs under the same source tree, we recommend running bazel clean when switching between these two configurations in the same source tree.

So, `build clean` should be run from the source tree. This is the folder where I had downloaded Tensorflow source code from GitHub. 

I ran `build clean` from that folder and it removed all those Tensorflow files from `bazel-out`. 

So, that's good. 

That `bazel-out` folder was present inside -

`C:\Users\UserName\_bazel_UserName`

And that `_bazel_UserName` folder still takes up about ~1.5GB of space. 

It's not so much about the space anymore as it is about removing Bazel from Windows, so, I will keep the issue open for now. Thanks!

**Comment by philwo** [comment]
Agreed, uninstall instructions that include removal steps for the output base folder would be nice.

**Comment by alkhwarizmi** [comment]
I also have problems removing Bazel. I had to install it because I needed to recompile tensorflow. In my case I installed through baselisk, which installed a version not reccommended for tensorflow. I would appreciate a lot if instruction could be found in the manual on how to remove bazel and baselisk from Windows.
